File: Go-Workspace/go-crud-demo/src/entitys/student.go
package entitys

import "gorm.io/gorm"

type Student struct {
	gorm.Model
	Name   string `gorm:"type:varchar(20); not null" json:"name" binding:"required"`
	Gender string `gorm:"type:varchar(20); not null" json:"gender" binding:"required"`
	Age    int    `gorm:"type:int; not null" json:"age" binding:"required"`
	Class  string `gorm:"type:varchar(20); not null" json:"class" binding:"required"`
	Score  int    `gorm:"type:int; not null" json:"score" binding:"required"`
}

File: java-workspace/demo/src/main/java/com/hjc/demo/pojo/Student.java
package com.hjc.demo.pojo;

import lombok.Data;
import org.springframework.stereotype.Component;

@Data
@Component
public class Student {
    private Integer id;
    private String name;
    private String gender;
    private Integer age;
    private String sClass;
    private Integer score;
//    private Account accountInfo;
}
